sunpy/visualization/tests/test_mapcubeanimator.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+import os
+
+import matplotlib.pyplot as plt
+import pytest
+
+import sunpy.map
+from sunpy.visualization import mapcubeanimator
+from sunpy.data import test
+from sunpy.tests.helpers import figure_test
+
+
+@pytest.fixture
+def map_animator():
+    testpath = test.rootdir
+    aia_file = os.path.join(testpath, "aia_171_level1.fits")
+    mapcube = sunpy.map.Map(aia_file)
```

Comment:
I would suggest making a mapcube with more than one image.

Reply:
Sure, will do that.